components/card/index.tsx

```diff
@@ -160,9 +162,19 @@ export default class Card extends React.Component<CardProps, {}> {
       </div>
     );
 
+    const extraProps = {
+      [activeTabKey ? 'activeKey' : 'defaultActiveKey']:
+        activeTabKey || defaultActiveTabKey,
```

Comment:
What about `activeTabKey={0}`?

Reply:
Sorry, I didn't understand what you meant.
The `activeTabKey` should be passed in a string type.
So how should `activeTabKey={0}` be handled?

Reply:
Use `activeTabKey === undefined` is more robust in case of  `activeTabKey=""` or else.

Reply:
Thank you for pointing, updated.